spec/support/session_helpers.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Support
+  module SessionHelpers
+    def sign_up_with(email, password)
+      visit new_user_registration_path
```

Comment:
Imagine you have 100, or 1000 tests that all depend on the fact that user is logged in. Doing all of that logging in through the UI in specs is going to take hours of unnecessary time. 

This is quite possibly overkill for that project, but in a production project I would say this:

- use the UI logic directly in your login/sign up tests
- Add a trait for your user factory called `logged_in: true`, which would create the user already logged in.

Feel free to either skip this or investigate, your call.